docs/background/energy_offset_array.rst

```diff
@@ -0,0 +1,24 @@
+.. _bg_models:
+
+EnergyOffset Array
+=================
```

Comment:
You should add `energy_offset_array` to the toctree directive here, so that it's possible to navigate to this new page:
https://github.com/gammapy/gammapy/blame/master/docs/background/index.rst#L41